mappings/net/minecraft/resource/ResourceReloadListener.mapping

```diff
@@ -1,9 +1,10 @@
 CLASS wz net/minecraft/resource/ResourceReloadListener
-	CLASS wz$a Helper
-		METHOD a waitForAll (Ljava/lang/Object;)Ljava/util/concurrent/CompletableFuture;
+	CLASS wz$a Anchor
```

Comment:
Just curious, what do you mean by `Anchor`?

Reply:
anchor is like a fixed point, like a spike. Idea came from mediawiki's anchor for link and railcraft's world anchor.

Reply:
Yes, I understand that. But how does it make sense for resource packs?

Reply:
for listeners, it is an anchor point so that the completable future for apply stage can be executed after this anchor point's completable future is done.